planex/init.py

```diff
@@ -0,0 +1,49 @@
+#!/usr/bin/env python
+
+"""
+Creates or regenerates a Makefile with special planex-init comments
+"""
+
+import os
+import logging
+
+MAKEFILE_PATH = "/usr/share/planex"
+
+log = logging.getLogger(__name__)
+
+
+def create_makefile():
+    """ Checks if a Makefile exists with special planex-init comments in it.
+    If not, it creates or regenerates the Makefile while preserving its
+    existing contents.
+    """
+    name = "Makefile"
+
+    firstline = "# Start generated by planex-init\n"
+    autogen = "include %s/Makefile.rules\n" % (MAKEFILE_PATH)
+    endline = "# End generated by planex-init\n"
+
+    if not os.path.exists(name):
+        log.debug("Creating Makefile")
+        with open(name, 'w') as h:
+            h.write(firstline)
+            h.write(autogen)
+            h.write(endline)
+
```

Comment:
You can exit(0) here, to avoid going through the rewriting procedure when you have just created the file.